format/index.md

```diff
@@ -1361,10 +1363,14 @@ return a `202 Accepted` status code.
 
 If a server accepts an update but also changes the resource(s) in other ways
 than those specified by the request (for example, updating the `updated-at`
-attribute or a computed `sha`), it **MUST** return a `200 OK` response.
+attribute or a computed `sha`), it **MUST** return a `200 OK` response. The
+response document **MUST** include a representation of the updated resource(s)
+as if a `GET` request was made to the request URL.
 
-The response document for a `200 OK` **MUST** include a representation of
-the updated resource(s) as if a `GET` request was made to the request URL.
+A server **MUST** return a `200 OK` status code if an update is successful,
```

Comment:
I would remove this section and add a special case in 204 stating that if meta is included, 200 must be used in lieu of 204.